sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -275,13 +275,28 @@ public static class HttpHeaders {
         // Client Encryption Headers
         public static final String IS_CLIENT_ENCRYPTED_HEADER = "x-ms-cosmos-is-client-encrypted";
         public static final String INTENDED_COLLECTION_RID_HEADER = "x-ms-cosmos-intended-collection-rid";
+
+        // SDK supported capacities headers
+        public static final String SDK_SUPPORTED_CAPABILITIES = "x-ms-cosmos-sdk-supported-capabilities";
     }
 
     public static class A_IMHeaderValues {
         public static final String INCREMENTAL_FEED = "Incremental Feed";
         public static final String FULL_FIDELITY_FEED = "Full-Fidelity Feed";
     }
 
+    public static class SDKSupportedCapabilities {
+        private static final long None = 0; // 0
```

Comment:
Since they are only being used as Strings, I am curious to know why we have declared them as long?